### PR TITLE
#44_tab_menu_wrong_id

### LINKED ISSUE #44: As a user, tab menu IDs are immediately correct
I think this has to do with a null pointer somewhere. I generally know what I want to fix here but I'm not sure if it will work.

### PR DESCRIPTION
**Pull request by solarisstar**
Can't believe how easy this one was. I tested it out with three clients connecting and disconnecting and it looked fine. 

Fixes #44

### CODE REVIEW

**Comment by solarisstar** [comment]
#44

File: code/cgame/cg_scoreboard.c [hunk 1]
```diff
@@ -329,21 +328,24 @@ static void CG_DrawClientScore_Big(int y, score_t *score, float *color, float fa
...
+        int player_id = Com_Clamp(0, 128, score->client);
+        Com_sprintf(id_string, sizeof id_string, "%i", player_id);
+        UI_DrawProportionalString(SB_SCORE_X_BIG, y, id_string, UI_TINYFONT | UI_LEFT, colorTable[CT_WHITE]);
 
         //player time
-        //If someone actually hits this limit, they officially have no life.
-        //Com_sprintf(string,sizeof(string),"%i", intClamp);
-        intClamp = Com_Clamp(0, 99999, score->time);
-        UI_DrawProportionalString(SB_TIME_X_BIG, y, va("%i", intClamp), UI_TINYFONT | UI_LEFT, colorTable[CT_WHITE]);
+        int player_tine = Com_Clamp(0, 99999, score->time);
```
Comment (MJLHThomassen) [inline code/cgame/cg_scoreboard.c]: player_tine -> player_time

Comment (solarisstar) [inline code/cgame/cg_scoreboard.c]: Good catch. I'll get that up tonight.

**Comment by solarisstar** [comment]
Fixed the typo.

**Pull request merged**